settings.gradle.kts

```diff
@@ -361,7 +362,12 @@ include(":instrumentation:kafka:kafka-clients:kafka-clients-0.11:javaagent")
 include(":instrumentation:kafka:kafka-clients:kafka-clients-0.11:testing")
 include(":instrumentation:kafka:kafka-clients:kafka-clients-2.6:library")
 include(":instrumentation:kafka:kafka-clients:kafka-clients-common-0.11:library")
+include(":instrumentation:kafka:kafka-clients:kafka-clients-common-0.11:testing")
+include(":instrumentation:kafka:kafka-connect-2.6:javaagent")
+include(":instrumentation:kafka:kafka-connect-2.6:testing")
+include(":instrumentation:kafka:kafka-streams-0.11:bootstrap")
 include(":instrumentation:kafka:kafka-streams-0.11:javaagent")
+include(":instrumentation:kotlinx-coroutines:javaagent")
```

Comment:
This PR seems to only add kafka-connect-2.6, so I'm surprised to see these changes include all these references in the topmost settings.gradle.kts. Any idea why or what I'm missing?